src/components/Home.vue

```diff
@@ -1,31 +1,21 @@
 <script setup>
-import { ref, defineProps, watchEffect } from "vue";
-import UserContributionForm from "./UserContributionForm.vue";
+// import { stringify } from "querystring";
+// import { ref } from "vue";
 
 const props = defineProps({
-  tasks: Array,
+  toolName: String,
+  toolDescription: String,
+  missingField: String,
+  toolURL: String,
   isError: Boolean,
-  currentUser: String,
 });
-const currentTaskIndex = ref(0);
-const currentTask = ref(null);
-
-watchEffect(() => {
-  currentTask.value = props.tasks[currentTaskIndex.value];
-});
-
-function getNextTask() {
-  currentTaskIndex.value++;
-  if (props.tasks.length <= currentTaskIndex.value) {
-    // for now it will start at 0 but when connected to backend we can fetch a batch of tasks
-    currentTaskIndex.value = 0;
-  }
-}
+// const currentTaskIndex = ref(0);
```

Comment:
No commented out code

Reply:
Addressed in [b3895cc](https://github.com/wikimedia/toolhunt-ui/pull/41/commits/b3895ccc14df0fc0d4bc2bec6257fa73d9fa4ecb)